studio/components/layouts/TableEditorLayout/TableEditorMenu.tsx

```diff
@@ -224,11 +200,35 @@ const TableEditorMenu = () => {
                 <>
                   <div className="flex w-full items-center justify-between">
                     <div className="flex items-center space-x-1">
-                      <p>Tables</p>
-                      {totalCount !== undefined && (
-                        <p style={{ fontVariantNumeric: 'tabular-nums' }}>({totalCount})</p>
-                      )}
+                      {/* Table search input */}
+                      <div className="mb-2 block relative">
+                        <div className={`relative table-editor-search border-none'}`}>
+                          {isSearchExpanded ? (
+                            <div className="flex items-center">
+                              <Input
```

Comment:
Probably want to always have an Input.

so rather than the SearchIcon being replaced with an Input, its actually just an Input with CSS classes changing on it.

Reply:
initially I figured you'd do something like:

<Input className="w-8 !border-transparent focus:w-full focus:border"/>

although that won't help for when the input becomes unfocused, but has a value.

so maybe 

<Input className={cn(
"w-8 !border-transparent",
searchText && "!w-full !border"
)}/>

Reply:
Feel free to use `Input_Shadcn_` component, might be easier. will be easier to control.

```jsx

import {Input_Shadcn_} from 'ui'
import {Search} from 'lucide-react'

<Input_Shadcn_ className={cn(
  "w-8 !border-transparent",
  searchText && "!w-full !border"
)}>
  <Search/>
</Input_Shadcn_>
```

Reply:
It doesnt look like we can include the `<Search/>` icon inside `Input_Shadcn_` component, I get a blank page.